sdcm/sct_events/setup.py

```diff
@@ -55,6 +56,9 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
     time.sleep(EVENTS_SUBSCRIBERS_START_DELAY)
 
     # Default filters.
+    workload_line = 'workload prioritization - update_service_levels_from_distributed_data: an error occurred while ' \
+                    'retrieving configuration'
+    EventsSeverityChangerFilter(new_severity=Severity.WARNING, event_class=DatabaseLogEvent, regex=workload_line)
```

Comment:
You forgot to `.publish()` this event.

Reply:
when i change to what you suggest above, i got this from lint:
`Expected type 'Optional[Type[SctEvent]]', got 'Type[LogEventProtocol]' instead`